website/docs/gs/wdls.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Pipeline
```

Comment:
This content also appears under `/website/docs/modules/index.md`

Reply:
I removed.